### PR TITLE
Add docs update actions

### PR DESCRIPTION
**Pull request by alex-ct**

### CODE REVIEW

**Comment by alex-ct** [comment]
I realise there are some docs missing. Reason is that MD files are not formatted the same, its hard to cover all variations.

File: types/cart-discount/updates/CartDiscountSetDescriptionAction.raml [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-
 type: CartDiscountUpdateAction
 displayName: CartDiscountSetDescriptionAction
 discriminatorValue: setDescription
 (postman-example): !include ../../../examples/CartDiscount/CartDiscountSetDescriptionAction.json
 properties:
   description?:
     type: LocalizedString
+    description: If the `description` parameter is not included, the field will be
+      emptied.
```
Comment (katmatt) [inline types/cart-discount/updates/CartDiscountSetDescriptionAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/cart-discount/updates/CartDiscountSetValidFromAction.raml [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 (docs-uri): https://docs.commercetools.com/http-api-projects-cartDiscounts.html#set-valid-from
-
 type: CartDiscountUpdateAction
 displayName: CartDiscountSetValidFromAction
 discriminatorValue: setValidFrom
 (postman-example): !include ../../../examples/CartDiscount/CartDiscountSetValidFromAction.json
 properties:
   validFrom?:
     type: datetime
+    description: If absent, the field with the value is removed in case a value was
```
Comment (katmatt) [inline types/cart-discount/updates/CartDiscountSetValidFromAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/cart-discount/updates/CartDiscountSetValidFromAndUntilAction.raml [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
 (docs-uri): https://docs.commercetools.com/http-api-projects-cartDiscounts.html#set-valid-from-and-until
-
 type: CartDiscountUpdateAction
 displayName: CartDiscountSetValidFromAndUntilAction
 discriminatorValue: setValidFromAndUntil
 (postman-example): !include ../../../examples/CartDiscount/CartDiscountSetValidFromAndUntilAction.json
 properties:
   validFrom?:
     type: datetime
+    description: If absent, the field with the value is removed in case a value was
```
Comment (katmatt) [inline types/cart-discount/updates/CartDiscountSetValidFromAndUntilAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/cart-discount/updates/CartDiscountSetValidFromAndUntilAction.raml [hunk 4]
```diff
@@ -1,13 +1,16 @@
...
+      set before.
   validUntil?:
     type: datetime
+    description: If absent, the field with the value is removed in case a value was
```
Comment (katmatt) [inline types/cart-discount/updates/CartDiscountSetValidFromAndUntilAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/cart-discount/updates/CartDiscountSetValidUntilAction.raml [hunk 5]
```diff
@@ -1,11 +1,12 @@
...
 (docs-uri): https://docs.commercetools.com/http-api-projects-cartDiscounts.html#set-valid-until
-
 type: CartDiscountUpdateAction
 displayName: CartDiscountSetValidUntilAction
 discriminatorValue: setValidUntil
 (postman-example): !include ../../../examples/CartDiscount/CartDiscountSetValidUntilAction.json
 properties:
   validUntil?:
     type: datetime
+    description: If absent, the field with the value is removed in case a value was
```
Comment (katmatt) [inline types/cart-discount/updates/CartDiscountSetValidUntilAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/category/updates/CategoryAddAssetAction.raml [hunk 6]
```diff
@@ -1,14 +1,16 @@
...
 discriminatorValue: addAsset
 (postman-example): !include ../../../examples/Category/CategoryAddAssetAction.json
 properties:
   asset:
     type: AssetDraft
+    description: ''
   position?:
     type: number
     format: int32
+    description: When specified, the value might be `0` and should be lower than the
```
Comment (katmatt) [inline types/category/updates/CategoryAddAssetAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/category/updates/CategorySetCustomTypeAction.raml [hunk 7]
```diff
@@ -1,13 +1,17 @@
...
 (docs-uri): https://docs.commercetools.com/http-api-projects-categories.html#set-custom-type
-
 type: CategoryUpdateAction
 displayName: CategorySetCustomTypeAction
 discriminatorValue: setCustomType
 (postman-example): !include ../../../examples/Category/CategorySetCustomTypeAction.json
 properties:
   type?:
     type: TypeResourceIdentifier
+    description: If absent, the custom
```
Comment (katmatt) [inline types/category/updates/CategorySetCustomTypeAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/discount-code/updates/DiscountCodeSetCartPredicateAction.raml [hunk 8]
```diff
@@ -1,11 +1,12 @@
...
 (docs-uri): https://docs.commercetools.com/http-api-projects-discountCodes.html#set-cart-predicate
-
 type: DiscountCodeUpdateAction
 displayName: DiscountCodeSetCartPredicateAction
 discriminatorValue: setCartPredicate
 (postman-example): !include ../../../examples/DiscountCode/DiscountCodeSetCartPredicateAction.json
 properties:
   cartPredicate?:
     type: string
+    description: If the `cartPredicate` parameter is not included, the field will
```
Comment (katmatt) [inline types/discount-code/updates/DiscountCodeSetCartPredicateAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/discount-code/updates/DiscountCodeSetDescriptionAction.raml [hunk 9]
```diff
@@ -1,11 +1,12 @@
...
 (docs-uri): https://docs.commercetools.com/http-api-projects-discountCodes.html#set-description
-
 type: DiscountCodeUpdateAction
 displayName: DiscountCodeSetDescriptionAction
 discriminatorValue: setDescription
 (postman-example): !include ../../../examples/DiscountCode/DiscountCodeSetDescriptionAction.json
 properties:
   description?:
     type: LocalizedString
+    description: If the `description` parameter is not included, the field will be
```
Comment (katmatt) [inline types/discount-code/updates/DiscountCodeSetDescriptionAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/discount-code/updates/DiscountCodeSetMaxApplicationsAction.raml [hunk 10]
```diff
@@ -10,3 +9,5 @@ properties:
   maxApplications?:
     type: number
     format: int64
+    description: If the `maxApplications` parameter is not included, the field will
```
Comment (katmatt) [inline types/discount-code/updates/DiscountCodeSetMaxApplicationsAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/discount-code/updates/DiscountCodeSetMaxApplicationsPerCustomerAction.raml [hunk 11]
```diff
@@ -10,3 +9,5 @@ properties:
   maxApplicationsPerCustomer?:
     type: number
     format: int64
+    description: If the `maxApplicationsPerCustomer` parameter is not included, the
```
Comment (katmatt) [inline types/discount-code/updates/DiscountCodeSetMaxApplicationsPerCustomerAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/discount-code/updates/DiscountCodeSetValidFromAction.raml [hunk 12]
```diff
@@ -1,11 +1,12 @@
...
 (docs-uri): https://docs.commercetools.com/http-api-projects-discountCodes.html#set-validfrom
-
 type: DiscountCodeUpdateAction
 displayName: DiscountCodeSetValidFromAction
 discriminatorValue: setValidFrom
 (postman-example): !include ../../../examples/DiscountCode/DiscountCodeSetValidFromAction.json
 properties:
   validFrom?:
     type: datetime
+    description: If absent, the field with the value is removed in case a value was
```
Comment (katmatt) [inline types/discount-code/updates/DiscountCodeSetValidFromAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/discount-code/updates/DiscountCodeSetValidFromAndUntilAction.raml [hunk 13]
```diff
@@ -1,13 +1,16 @@
...
 (docs-uri): https://docs.commercetools.com/http-api-projects-discountCodes.html#set-valid-from-and-until
-
 type: DiscountCodeUpdateAction
 displayName: DiscountCodeSetValidFromAndUntilAction
 discriminatorValue: setValidFromAndUntil
 (postman-example): !include ../../../examples/DiscountCode/DiscountCodeSetValidFromAndUntilAction.json
 properties:
   validFrom?:
     type: datetime
+    description: If absent, the field with the value is removed in case a value was
```
Comment (katmatt) [inline types/discount-code/updates/DiscountCodeSetValidFromAndUntilAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/discount-code/updates/DiscountCodeSetValidUntilAction.raml [hunk 14]
```diff
@@ -1,11 +1,12 @@
...
 (docs-uri): https://docs.commercetools.com/http-api-projects-discountCodes.html#set-validuntil
-
 type: DiscountCodeUpdateAction
 displayName: DiscountCodeSetValidUntilAction
 discriminatorValue: setValidUntil
 (postman-example): !include ../../../examples/DiscountCode/DiscountCodeSetValidUntilAction.json
 properties:
   validUntil?:
     type: datetime
+    description: If absent, the field with the value is removed in case a value was
```
Comment (katmatt) [inline types/discount-code/updates/DiscountCodeSetValidUntilAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/product-type/updates/ProductTypeSetKeyAction.raml [hunk 15]
```diff
@@ -1,11 +1,14 @@
...
 type: ProductTypeUpdateAction
 displayName: ProductTypeSetKeyAction
 discriminatorValue: setKey
-
 properties:
   key?:
     type: string
+    description: |-
+      If `key` is absent or `null`, this field will be removed if it exists.
+      Min.
```
Comment (katmatt) [inline types/product-type/updates/ProductTypeSetKeyAction.raml]: This looks a bit weird...

File: types/product/updates/ProductAddAssetAction.raml [hunk 16]
```diff
@@ -10,12 +9,17 @@ properties:
...
   staged?:
     type: boolean
+    description: ''
   asset:
     type: AssetDraft
+    description: ''
   position?:
     type: number
     format: int32
+    description: Position of the new asset inside the existing list (from `0` to the
```
Comment (katmatt) [inline types/product/updates/ProductAddAssetAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/product/updates/ProductChangeSlugAction.raml [hunk 17]
```diff
@@ -1,13 +1,17 @@
...
 displayName: ProductChangeSlugAction
 discriminatorValue: changeSlug
 (postman-example): !include ../../../examples/Product/ProductChangeSlugAction.json
 properties:
   slug:
     type: LocalizedString
+    description: |-
+      Every slug must be unique across a project, but a product can have the same slug for different languages.
+      Allowed are alphabetic, numeric, underscore (`_`) and hyphen (`-`) characters.
+      Maximum size is {{ site.data.api-limits.slugLength }}.
```
Comment (katmatt) [inline types/product/updates/ProductChangeSlugAction.raml]: Here we should put in the actual value. This is a good example why parsing the HTML version might have been a better idea 😉 😁

File: types/project/updates/ProjectSetExternalOAuthAction.raml [hunk 18]
```diff
@@ -1,11 +1,11 @@
...
 (docs-uri): https://docs.commercetools.com/http-api-projects-project.html#set-externaloauth
-
 type: ProjectUpdateAction
 displayName: ProjectSetExternalOAuthAction
 discriminatorValue: setExternalOAuth
-
 properties:
   externalOAuth?:
     type: ExternalOAuth
+    description: If you do not provide the `externalOAuth` field or provide a value
```
Comment (katmatt) [inline types/project/updates/ProjectSetExternalOAuthAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/review/updates/ReviewSetTitleAction.raml [hunk 19]
```diff
@@ -1,11 +1,12 @@
...
 (docs-uri): https://docs.commercetools.com/http-api-projects-reviews.html#set-title
-
 type: ReviewUpdateAction
 displayName: ReviewSetTitleAction
 discriminatorValue: setTitle
 (postman-example): !include ../../../examples/Review/ReviewSetTitleAction.json
 properties:
   title?:
     type: string
+    description: If `title` is absent or `null`, this field will be removed if it
```
Comment (katmatt) [inline types/review/updates/ReviewSetTitleAction.raml]: This description is multi line, shouldn't it then start with `|`?

File: types/cart-discount/updates/CartDiscountSetDescriptionAction.raml (hunk 1 above)
Comment (alex-ct) [reply types/cart-discount/updates/CartDiscountSetDescriptionAction.raml]: Parser checks passed so I guess it ok to have it without | .

**Pull request merged**